contracts/keep3r/GenericKeep3rV2.sol

```diff
@@ -115,14 +130,24 @@ contract GenericKeep3rV2 is Governable, CollectableDust, Keep3r, IStrategyKeep3r
         }
     }
 
+    function harvestable2(address _strategy) public view returns (uint256) {
+        uint256 gasPrice = uint256(IChainLinkFeed(fastGas).latestAnswer());
+        uint256 callCost = requiredHarvest[_strategy].mul(gasPrice).mul(keep3rMaxMultiplier).div(10);
+        return callCost;
+    }
+
     function harvestable(address _strategy) public view override returns (bool) {
         require(requiredHarvest[_strategy] > 0, "generic-keep3r-v2::harvestable:strategy-not-added");
-        return IBaseStrategy(_strategy).harvestTrigger(requiredHarvest[_strategy]);
+        uint256 gasPrice = uint256(IChainLinkFeed(fastGas).latestAnswer());
+        uint256 callCost = requiredHarvest[_strategy].mul(gasPrice).mul(keep3rMaxMultiplier).div(10);
```

Comment:
`requiredHarvest[_strategy]` will contain the cost of the call, we multiply by the gas price and we assume the max multiplier for the keep3r fee.

Reply:
Assuming you meant "This will send the `callCost` in wei" not "ether", right?

Also, BaseStrategy will have to have these updated:
https://github.com/iearn-finance/yearn-vaults/blob/master/contracts/BaseStrategy.sol#L277
https://github.com/iearn-finance/yearn-vaults/blob/master/contracts/BaseStrategy.sol#L304

Will it be feasible for all strategies to convert this callCost to want? Wouldn't this require a WANT-ETH on-chain oracle? Are we planning to use Andre's UniswapSlidingWindowOracle or something?

Reply:
> Assuming you meant "This will send the callCost in wei" not "ether", right?
Yes! my bad. ethereum in wei

Reply:
> Will it be feasible for all strategies to convert this callCost to want? Wouldn't this require a WANT-ETH on-chain oracle? Are we planning to use Andre's UniswapSlidingWindowOracle or something?

We are still discussing it.
I believe a want oracle at the vault is the solution, we will see.